tools/sof-kernel-log-check.sh

```diff
@@ -179,7 +179,7 @@ ignore_str="$ignore_str"'|iwlwifi 0000:00:14\.3: Microcode SW error detected\. R
 ignore_str="$ignore_str"'|: authentication with ..:..:..:..:..:.. timed out'
 
 # Dell TGL laptop with SoundWire, issues reported by sof-test
-ignore_str="$ignore_str"'|ACPI BIOS Error (bug):'
+ignore_str="$ignore_str"'|ACPI BIOS Error \(bug\):'
```

Comment:
Normally, there should be more info after the `:`, eg, `ACPI BIOS Error (bug): Could not resolve symbol [\_SB.PC00.PEG0._PS3.HPEX]`, I think `ACPI BIOS Error (bug):` match exactly "ACPI BIOS Error (bug):", maybe add a `*` ?

Reply:
@aiChaoSONG I think we did not need that, this is already a regex.